sass/blocks/intro/_intro.scss

```diff
@@ -32,12 +43,31 @@
   &--smaller {
     height: 16rem;
     background: url("../img/text-sedona.svg"),
+      url("../img/bg-intro-triangle-mobile.svg"),
       url("../img/bg-back-mobile@1x.jpg");
     background-repeat: no-repeat,
+      no-repeat,
       no-repeat;
     background-size: 23.1rem,
+      100% 100%,
       auto;
     background-position: 49% 43%,
+      0 6.7rem,
       50% 100%;
+
+    @media (min-resolution: 144dpi), (min-resolution: 1.5dppx) {
+      background: url("../img/text-sedona.svg"),
```

Comment:
Move text-sedona.svg to the level of html, and get remove this third @media (min-resolution: 144dpi), (min-resolution: 1.5dppx) {
looks like smaller should only have height setting.